### ISSUE TITLE
Add title to preset themes

### ISSUE THREAD

**Issue by pdehaan** [mozilla/FirefoxColor#195]
When hovering over a preset theme, we should probably set a `title` attribute with the theme's title.
In fact, it looks like we do, but it isn't showing up for me locally, or on the github.io site.

https://github.com/mozilla/Themer/blob/413aa25b78865392779fdb680735d52d32568d03/src/web/lib/components/PresetThemeSelector/index.js#L15

Not sure if we need to tweak `normalizeTheme()` to possibly account for a title...
https://github.com/mozilla/Themer/blob/413aa25b78865392779fdb680735d52d32568d03/src/lib/themes.js#L45-L67

Not sure how we could add a meaningful `title` to the saved themes since I don't think we have the UI currently for the user to set a custom title. And I don't how how useful a generated title would be. We could do something boring like `title="Saved theme 3", but not sure how useful that'd be for a11y or screen readers. Especially since I think the saved themes get saved to the front of the queue so your numbering would always change by 1 when you add a new theme.

**Comment by johngruen** [comment]
yeah, i kinda want to do this, but i think we need real names for themes...like "Miami Freakout" or "Quato's Revenge"

**Comment by csd713** [comment]
Hi @pdehaan I would like to work on this. Do we have the list of names available for presets ?

**Comment by lmorchard** [comment]
I don't think this one is quite ready to work on yet - we need to come up with names and figure out how / where to display them on the page

**Comment by pdehaan** [comment]
@csd713 Unfortunately, we still need to do a bit of work before we can start on this.
But the good news is that we have a few other [good first issues](https://github.com/mozilla/FirefoxColor/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) you could take a look at.
